src/ResourceManager/Compute/Commands.Compute/AvailabilitySets/GetAzureAvailabilitySetCommand.cs

```diff
@@ -39,6 +39,7 @@ public class GetAzureAvailabilitySetCommand : AvailabilitySetBaseCmdlet
             Position = 1,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The availability set name.")]
+        [ResourceNameCompleter("Microsoft.Compute/availabilitySets", new string[] { "ResourceGroupName" })]
```

Comment:
Darnit. I didn't look at the common changes in correlation with how it is used. You should change the constructor definition to this instead in common:
```cs
public ResourceNameCompleterAttribute(string resourceType, params string[] parentResourceParameterNames)
```

Then, the usage (in all of these cases) becomes:
```cs
[ResourceNameCompleter("Microsoft.Compute/availabilitySets", "ResourceGroupName")]
```

Reply:
Yes, in this case, you will be able to use 

```c#
[ResourceNameCompleter(<typename>, <parentresource1>, <[parentResource2>, ...)]
```

Reply:
Fix is here: https://github.com/Azure/azure-powershell-common/pull/78